### ISSUE TITLE
Error 'ProgrammingError: no results to fetch' under 'NodeConnection.execute' method call that executes utility query that doesn't return any result

### ISSUE THREAD

**Issue by maksm90** [postgrespro/testgres#12]
This fragment:

```
with node.connect() as con:
      con.execute('create table range_partitioned as select generate_series(1, 1e4) i')
```

fails with error:

```
Traceback (most recent call last):
  ...
  File ".../lib/python2.7/site-packages/testgres/testgres.py", line 132, in execute
    return self.cursor.fetchall()
ProgrammingError: no results to fetch
```

**Comment by vadv** [comment]
fetchall() must return rows. Need to implement [PERFORM](https://www.postgresql.org/docs/9.6/static/plpgsql-statements.html)  analog.

**Comment by funbringer** [comment]
fixed in 0.1.15
